components/scream/scripts/machines_specs.py

```diff
@@ -16,49 +16,49 @@
 
 MACHINE_METADATA = {
     "melvin"   : (["module purge", "module load sems-env", "module load sems-gcc/7.3.0 sems-openmpi/1.10.1 sems-gcc/7.3.0 sems-git/2.10.1 sems-cmake/3.12.2 sems-python/3.5.2"],
-                  "$(which mpicxx)",
+                  "$(which mpicxx) -c CMAKE_Fortran_COMPILER=$(which mpifort) -c CMAKE_C_COMPILER=$(which mpicc)",
```

Comment:
@AaronDonahue , I think with @bartgol 's recent improvements to test_all_scream, these changes are no longer necessary. You should upstream-merge latest master into your PR, remove these cmake changes, and then confirm that things still work.

Reply:
@jgfouca , Ok, I rebased my branch and was able to confirm that things work for syrah (and I'm guessing quartz too).  But Lassen also needs `$(which mpicc)` which isn't a part of the PR that Luca put in.  Any thoughts on how to move forward?  Should we add mpicc in the same way we did mpifort?  Or have an extra option to pass other compiler flags for the test?  At which point I can just add the `-c CMAKE_C_COMPILER=$(which mpicc)` line.

Reply:
@bartgol ^^

Reply:
@AaronDonahue , if we are compiling C files then we probably need to add explicit C support just as @bartgol did for f90. For now, is there a way you can hardcode Lassen to add what you need? If not, I can quickly repeat what Luca did for f90 for C.

Reply:
I may be able to hardcode it for lassen for now.  But it may be good to expand support for C if possible, just in case other machines have a similar requirement.

Reply:
Should this PR take care of mpicc support, or should it be another PR?

Reply:
It should probably be another PR.  This PR is getting pretty big.  On a side note, I am planning a small PR for machine support that will allow me to get the >1 mpi rank tests to work on syrah, quartz and lassen.  I could put that in and then @bartgol or @jgfouca you could add the mpicc support to it if you want.

Reply:
Also, for lassen I just added the mpicc support to the lassen.cmake file.  So if my PR is just left as is we will have the lassen tests running and passing without any extra changes to test-all-scream.

Reply:
@jgfouca I can do the mpicc one. Should be a 2 min thing.